MiniTwitch.Irc/Models/Privmsg.cs

```diff
@@ -252,7 +252,7 @@ internal Privmsg(ReadOnlyMemory<byte> memory, IrcClient? source = null)
 
                 //pinned-chat-paid-level
                 case 2139:
-                    level = TagHelper.GetString(tagValue, intern: true);
+                    level = TagHelper.GetEnum<HypeChatLevel>(tagValue);
```

Comment:
I don't suppose Intellenum can read values from a ReadOnlySpan<byte>? Even if it did, the perf gain of new nanoseconds wouldn't be enough for me to justify adding a dependency. I will keep that in mind though, in case it's needed in the future 👍

Reply:
Yeah, I only implemented `ReadOnlySpan<char>`. I could open an issue with Steve and discuss its usefulness - but there would be some other things in Intellenum that would need to be changed before that.